google-cloud-spanner/src/test/resources/com/google/cloud/spanner/connection/ITSqlScriptTest_TestQueryOptions.sql

```diff
@@ -43,7 +46,21 @@ SET OPTIMIZER_VERSION = '';
 @EXPECT RESULT_SET 'OPTIMIZER_VERSION',''
 SHOW VARIABLE OPTIMIZER_VERSION;
 
+-- Optimizer statistics package
+@EXPECT NO_RESULT
+SET OPTIMIZER_STATISTICS_PACKAGE = 'custom-package_withNumbers-1234';
+
+@EXPECT RESULT_SET 'OPTIMIZER_STATISTICS_PACKAGE','custom-package_withNumbers-1234'
+SHOW VARIABLE OPTIMIZER_STATISTICS_PACKAGE;
+
+@EXPECT NO_RESULT
+SET OPTIMIZER_STATISTICS_PACKAGE = '';
+
+@EXPECT RESULT_SET 'OPTIMIZER_STATISTICS_PACKAGE',''
+SHOW VARIABLE OPTIMIZER_STATISTICS_PACKAGE;
+
 -- Try to set invalid values.
+-- Optimizer version
 @EXPECT EXCEPTION INVALID_ARGUMENT 'INVALID_ARGUMENT: Unknown value for OPTIMIZER_VERSION: 'None''
 SET OPTIMIZER_VERSION = 'None';
 
```

Comment:
Should we add test statements for invalid values for optimizer package as well?

Reply:
That is a good point. Currently, I have not found any invalid scenarios, so I am asking on the design documentation if the regex I have set is the correct one. Once I have confirmation I will add the tests.

Reply:
Fixed in e2d6b2c.